Source/Scene/Cesium3DTileset.js

```diff
@@ -1358,11 +1341,8 @@ Object.defineProperties(Cesium3DTileset.prototype, {
 
   /**
    * A custom shader to apply to all tiles in the tileset. Only used for
-   * contents that use {@link ModelExperimental}. Using custom shaders with a
+   * contents that use {@link Model}. Using custom shaders with a
    * {@link Cesium3DTileStyle} may lead to undefined behavior.
```

Comment:
Is there a type of content that doesn't use `Model`? If not, remove the "Only used..." sentence

Reply:
vector and geometry contents don't use the new `Model` at present.